randovania/games/samus_returns/logic_database/header.txt

```diff
@@ -35,37 +35,266 @@ Templates
+                                      All of the following:
+                                          Aeion ≥ 1000
+                                          Any of the following:
+                                              Missile ≥ 21
+                                              Super Missile ≥ 6 and Shoot Super Missile
+                              All of the following:
+                                  Plasma Beam and Enabled Highly Dangerous Logic
+                                  Any of the following:
+                                      All of the following:
+                                          Aeion ≥ 2200
+                                          Any of the following:
+                                              Missile ≥ 8
+                                              Super Missile ≥ 2 and Shoot Super Missile
+                                      All of the following:
+                                          Aeion ≥ 1500
+                                          Any of the following:
+                                              Missile ≥ 12
+                                              Super Missile ≥ 5 and Shoot Super Missile
+                                      All of the following:
+                                          Aeion ≥ 1000
+                                          Any of the following:
+                                              Missile ≥ 25
+                                              Super Missile ≥ 7 and Shoot Super Missile
+
+* Defeat Zeta Metroid+:
+      All of the following:
+          Any of the following:
+              # Energy Requirements
+              Combat (Advanced) or Normal Damage ≥ 299
+              Combat (Beginner) and Normal Damage ≥ 199
+              Combat (Intermediate) and Normal Damage ≥ 100
+          Any of the following:
+              All of the following:
+                  Grapple Beam
+                  Missile or Shoot Super Missile
+              All of the following:
+                  Combat (Advanced)
+                  Any of the following:
+                      Missile ≥ 55
+                      Super Missile ≥ 14 and Shoot Super Missile
+              All of the following:
+                  Metroids without Missiles (Advanced)
+                  Any of the following:
+                      Ice Beam
+                      All of the following:
+                          Beam Burst
+                          Any of the following:
+                              All of the following:
+                                  Shoot Spazer Beam
+                                  Any of the following:
+                                      All of the following:
+                                          Aeion ≥ 2200
+                                          Any of the following:
+                                              Missile ≥ 8
+                                              Super Missile ≥ 2 and Shoot Super Missile
```

Comment:
What is this monstrosity ? The nested level is `All` `Any` `All` `Any` `All` `Any` `All` `Any` `All` `Any`
I tried to understand it 3 times without success. So, I just approve xD

Reply:
Something totally overkill lol. Just accounting for beam Burst usage with differing ammo counts